pkg/tarsum/tarsum_test.go

```diff
@@ -251,6 +291,33 @@ var (
 	sha512Hash = NewTHash("sha512", sha512.New)
 )
 
+// Test all the build-in read size : buf8K, buf16K, buf32K and more
+func TestTarSumsReadSize(t *testing.T) {
+	// Test this on the four first layer
+	for i := 1; i < 6; i++ {
```

Comment:
Actually I just want to loop 5 times, I'm always reader the same *layer* as I'm just testing the different "read size".